pyproject.toml

```diff
@@ -1,52 +1,36 @@
-[build-system]
-requires = ["setuptools"]
-build-backend = "setuptools.build_meta"
-
-[project]
+[tool.poetry]
 name = "imap-data-access"
 version = "0.2.0"
 description = "IMAP SDC Data Access"
-authors = [{name = "IMAP SDC Developers", email = "imap-sdc@lists.lasp.colorado.edu"}]
+authors = ["IMAP SDC Developers <imap-sdc@lists.lasp.colorado.edu>"]
+license = "MIT"
 readme = "README.md"
-license = {text = "MIT"}
-keywords = ["IMAP", "SDC", "SOC", "SDS", "Science Operations"]
-classifiers = [
-    "Development Status :: 3 - Alpha",
-    "License :: OSI Approved :: MIT License",
-    "Natural Language :: English",
-    "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3.9",
-    "Programming Language :: Python :: 3.10",
-    "Programming Language :: Python :: 3.11",
-    "Programming Language :: Python :: 3.12",
-    "Topic :: Software Development",
-    "Topic :: Scientific/Engineering",
-    "Operating System :: Microsoft :: Windows",
-    "Operating System :: POSIX",
-    "Operating System :: Unix",
-    "Operating System :: MacOS",
-]
-
-[project.scripts]
-imap-data-access = "imap_data_access.cli:main"
 
-[project.urls]
-homepage = "https://github.com/IMAP-Science-Operations-Center"
-repository = "https://github.com/IMAP-Science-Operations-Center/imap-data-access"
+[tool.poetry.dependencies]
+python = "^3.9"
+pre-commit = "^3.6.0"
 
+[tool.poetry.group.dev.dependencies]
+ruff = "^0.2.1"
 
-[project.optional-dependencies]
-dev = ["ruff"]
-test = ["pytest", "pytest-cov"]
+[tool.poetry.group.test.dependencies]
+pytest = "^8.0.0"
+pytest-cov = "^4.1.0"
 
+[build-system]
+requires = ["poetry-core"]
+build-backend = "poetry.core.masonry.api"
 
 [tool.pytest.ini_options]
 testpaths = [
   "tests",
 ]
 addopts = "-ra"
-
-[tool.ruff]
-target-version = "py39"
-select = ["B", "E", "F", "I", "N", "W", "PL", "PT", "UP", "RUF"]
-ignore = ["PLR2004"]
+markers = [
```

Comment:
We should leave the ruff section and remove the unused pieces that were added here.